src/controllers/EventsController.php

```diff
@@ -666,4 +684,33 @@ public function rejectAction($request, $db)
 
         return;
     }
+
+
+    public function setEventMapper(EventMapper $event_mapper)
+    {
+        $this->event_mapper = $event_mapper;
+    }
+
+    public function getEventMapper($db, $request)
+    {
+        if (! isset($this->event_mapper)) {
+            $this->event_mapper = new EventMapper($db, $request);
+        }
+
+        return $this->event_mapper;
+    }
+
+    public function setPendingTalkClaimMapper(PendingTalkClaimMapper $pending_talk_claim_mapper)
+    {
+        $this->pending_talk_claim_mapper = $pending_talk_claim_mapper;
+    }
+
+    public function getPendingTalkClaimMapper($db, $request)
+    {
+        if (! isset($this->pending_talk_claim_mapper)) {
+            $this->pending_talk_claim_mapper = new PendingTalkClaimMapper($db, $request);
+        }
+
+        return $this->pending_talk_claim_mapper;
+    }
```

Comment:
I agree, however I don't think it should form part of this PR. With ZendCon this week, I think we are well and truly in 'Conference Season', so I think getting this claim function live needs to be a priority.

That said, if we are going to aim to get test coverage up, we certainly need to do something, either DI or Factories - fancy raising a JIRA ticket? :-)